### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by jkomendant**

### CODE REVIEW

File: src/test/java/FC/StudyControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    public void getNextToStudy() throws SQLException {
+        //given
+        String deckName = "Sample";
+        StudyController studyController = new StudyController(deckName);
+
+        //when
+        Card card = studyController.getNextToStudy();
+
+        //then
+        assert (card.getDef() != null);
```
Comment (rpeikes) [inline src/test/java/FC/StudyControllerTest.java]: change to assertNotNull

File: src/test/java/FC/StudyControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        String deckName = "Sample";
+        StudyController studyController = new StudyController(deckName);
+        Card card = studyController.getDeck(deckName).getCard(1);
+        Deck deck = studyController.getUnMastered();
+
+        //when
+        studyController.masterCard(card);
+
+        //then
+        assert (deck.getSize() == 4);
```
Comment (rpeikes) [inline src/test/java/FC/StudyControllerTest.java]: assertEquals

**Review by rpeikes — CHANGES_REQUESTED** [review]
Great work

File: src/test/java/FC/StudyControllerTest.java (hunk 1 above)
Comment (jkomendant) [reply src/test/java/FC/StudyControllerTest.java]: Done

File: src/test/java/FC/StudyControllerTest.java (hunk 2 above)
Comment (jkomendant) [reply src/test/java/FC/StudyControllerTest.java]: Done

**Review by rpeikes — APPROVED** [review]
Great work, Jennifer!

**Pull request merged**